nautobot/docs/development/model-features.md

```diff
@@ -0,0 +1,40 @@
+# Registry Model Features
+
+A dictionary of particular features (e.g. custom fields) mapped to the Nautobot models which support them, arranged by app.
```

Comment:
Seems like it'd be logical to document `@extras_features` in this same file, unless we're planning to remove it entirely before 2.0.0 ships?

Reply:
I don't think there is a plan on removing `@extra_features`? @nautobot/core.

Im not also sure there is any detailed documentation for `@extra_features` currently existing in our docs

Reply:
That was my point :) We don't have a good document for that but if it's going to stick around we *should* document it, and this would be the logical file to do so in. Not blocking for this PR though.